java-agent/src/nativeMain/kotlin/com/epam/drill/agent/AgentArguments.kt

```diff
@@ -27,8 +27,9 @@ data class AgentArguments(
     val buildVersion: String? = null,
     val instanceId: String = "",
     val groupId: String = "",
-    val logLevel: String = LogLevel.ERROR.name,
+    val logLevel: String = KotlinLoggingLevel.ERROR.name,
     val logFile: String? = null,
+    val logLimit: Int = 512,
```

Comment:
`512` is too short and will trim most of the valuable info
I suggest setting default `logLimit` to 4000